### PR TITLE
Lab 4 notes

### PR DESCRIPTION
**Pull request by rmartin-sc**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -44,20 +44,26 @@ <h1>Welcome to Ashley's Website!</h1>
             
             <table>
                 <caption>Table 1. Current Courses</caption>
-                <tr><th rowspan="2">Term</th><th colspan="3">Course</th></tr>
-                <tr><th>Code</th><th>Name</th><th>Hrs/Week</th></tr>
-                <tr><th rowspan="6">Fall</th><td>CSD105</td><td>Python</td><td>3</td></tr>
+                <thead>
```
Comment (rmartin-sc) [inline index.html:47]: You need to add `<thead>` `<tbody>` and `<tfoot>` explicitly if you include more than just `<tbody>` as `<tbody>` is only added automatically

File: index.html [hunk 2]
```diff
@@ -44,20 +44,26 @@ <h1>Welcome to Ashley's Website!</h1>
...
+                <tr><th rowspan="2" scope="col">Term</th><th colspan="3" scope="colgroup">Course</th></tr>
+                <tr><th scope="col">Code</th><th scope="col">Name</th><th scope="col">Hrs/Week</th></tr>
```
Comment (rmartin-sc) [inline index.html:49]: You forgot to include header cell scopes, as per lab instructions (1.3.3)

File: survey.html [hunk 3]
```diff
@@ -39,15 +39,15 @@ <h1>Survey</h1>
             >
                 
 
-                <p><label for="fname">What is your name?</label>
+                <p><label for="fname">What is your name?<abbr title="required" aria-label="required">*</abbr></label>
```
Comment (rmartin-sc) [inline survey.html:42]: Whoops!  You missed the instruction 3.3!

File: survey.html [hunk 4]
```diff
@@ -70,20 +70,20 @@ <h1>Survey</h1>
                          <label for="no">No</label>
                         </li>
                         <li>
-                         <input type="radio" id="none" name="radio_button" value="no" checked>
+                         <input type="radio" id="none" name="radio_button" value="?" checked>
```
Comment (rmartin-sc) [inline survey.html:73]: Whoops!

File: survey.html [hunk 5]
```diff
@@ -70,20 +70,20 @@ <h1>Survey</h1>
...
                 
-                <br>
+                <p>
                 <label for="txt_area">What is the meaning of life?</label> 
                 <textarea id="txt_area" name="life_meaning" cols="50" rows="5" placeholder="Enter the correct answer here"></textarea>
-                 
-                <br>
-                <br>
-                    <input type="submit" id="send_btn" value="Send">
-                    <Label for="send_btn"></Label>
```
Comment (rmartin-sc) [inline survey.html:86]: No need for labels elements with buttons, as they have an implicit label set by the content of the element

File: survey.html [hunk 6]
```diff
@@ -70,20 +70,20 @@ <h1>Survey</h1>
...
                         <li>
-                         <input type="radio" id="none" name="radio_button" value="no" checked>
+                         <input type="radio" id="none" name="radio_button" value="?" checked>
                          <label for="none">None of your business</label>
                         </li>
                      </ul>
                 </fieldset>
                 
-                <br>
+                <p>
```
Comment (rmartin-sc) [inline survey.html:79]: Use `<p>` as per lab instructions

Comment (ashixiv) [reply survey.html:79]: I thought its okay to put <br> to have a break in radio button & txt area. You said that its better to put <br> but p is fine for now.. My bad i didnt read the instruction well.

**Pull request merged**